api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -184,6 +193,27 @@ COUNT(distinct p1.PERSON_ID) as count_value,COUNT(distinct p1.PERSON_ID) as sour
 on p1.person_id = co1.person_id where co1.condition_concept_id != co1.condition_source_concept_id
 group by co1.condition_source_concept_id, p1.gender_concept_id"
 
+# Condition gender identity
+# Calculating the gender identity counts for condition concepts based on the answer of each person to ppi question 1585838
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+"insert into \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\`
+(id, analysis_id, stratum_1, stratum_2, stratum_3, count_value, source_count_value)
+select 0, 3107 as analysis_id,
+	CAST(co1.condition_concept_id AS STRING) as stratum_1,
+	CAST(p1.gender_identity_concept_id AS STRING) as stratum_2,'Condition' as stratum_3,
+	COUNT(distinct p1.PERSON_ID) as count_value,
+	(select COUNT(distinct p2.PERSON_ID) from \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.person_gender_identity\` p2 inner join \`${BQ_PROJECT}.${BQ_DATASET}.condition_occurrence\` co2 on p2.person_id=co2.person_id where co2.condition_source_concept_id=co1.condition_concept_id and p2.gender_identity_concept_id = p1.gender_identity_concept_id) as source_count_value
```

Comment:
nit: can this line be broken up and indented (here and below)? This is difficult to read

Reply:
Alright, pushed in the change. Thanks.